src/lib/bcmp/dfu/bm_dfu_client.cpp

```diff
@@ -307,7 +303,8 @@ void s_client_receiving_entry(void) {
     bm_dfu_req_next_chunk(client_ctx.host_node_id, client_ctx.current_chunk);
 
     /* Kickoff Chunk timeout */
-    configASSERT(xTimerStart(client_ctx.chunk_timer, 10));
+    // configASSERT(xTimerStart(client_ctx.chunk_timer, 10));
+    bm_timer_start(client_ctx.chunk_timer, 10);
```

Comment:
I left most of the config asserts in for now because these take place inside of states within the statemachine and the error cannot be returned up to the statemachine (as it exists right now).

But I was thinking perhaps in the future we could utilize a similar thing to the `bm_err_check` macro for asserts where we assert if the user has set some flag for using asserts, or just print an error message. Not super urgent, but I thought leaving in the asserts might be a good reminder to do something.

Reply:
Is it worthwhile to place a TODO or something at the top of the header file or this file?

Reply:
Sure I will add that!